source/common/secret/secret_manager_impl.cc

```diff
@@ -65,37 +65,57 @@ SecretManagerImpl::createInlineCertificateValidationContextProvider(
 }
 
 void SecretManagerImpl::removeDynamicSecretProvider(const std::string& map_key) {
-  ENVOY_LOG(debug, "Unregister secret provider. hash key: {}", map_key);
+  ENVOY_LOG(debug, "Unregister tls certificate provider. hash key: {}", map_key);
```

Comment:
Is this definitely TLS? It seems the general `SdsApi` type of `dynamic_secret_providers_` might suggest otherwise?

Reply:
Yes, this message is not correct. Changed to secret provider.